.github/workflows/scripts/biome_disable_check.py

```
@@ -25,28 +24,28 @@
 import re
 import argparse
 import sys
+from typing import List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)